valpas-web/test/integrationtests/homeview.test.ts

```diff
@@ -1,11 +1,23 @@
 import { textEventuallyEquals } from "../integrationtests-env/browser/content"
+import {
+  disableFeature,
+  resetFeatures,
+} from "../integrationtests-env/browser/core"
 import {
   dataTableEventuallyEquals,
   dataTableHeadersEventuallyEquals,
 } from "../integrationtests-env/browser/datatable"
 import { loginAs } from "../integrationtests-env/browser/reset"
 
 describe("Etusivun väliaikainen näkymä", () => {
+  beforeAll(() => {
```

Comment:
hmm, en kyllä ihan ymmärrä, miksei tämän testin pitäisi toimia myös maksuttomuusfeaturen kanssa? Edelleen kuitenkin kunta- ja seurantakäyttäjät pitäisi ohjata väliaiaikaiselle etusivulle.

Voisi vaan muuttaa testikäyttäjät sellaisiksi, että heillä ei ole maksuttomuusoikeuksia?

Reply:
hyvä näin: parempi, että testataan nimenomaan ilman feature-flagiä, kunnes homma tuotannossa. Lisäsin lapun taululle, että muistetaan tämä päivittää.